packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -267,6 +282,30 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
     );
   }
 
+  /// Adds the appropriate opacity under the magnifier if the background
+  /// color is transparent (and therefore are not adding in a gradient).
+  Widget _buildUnderMagnifierScreen() {
```

Comment:
I'm not sure I understand what's the visual effect we're going for here. It's to keep the alpha doubled up inside the lens if we're doubling up with a screen outside the lens too?

Reply:
Exactly. It looks a lot better with this feature.

Reply:
Oh ok. Make this check on line 334 instead. In general, don't add more things to the tree and then make them no-op with blank containers (especially for exceptional cases) since they add weight.